integrations/aircall/.nango/schema.ts

```
@@ -1,25 +1,35 @@
 // You can version this file
 // ---------------------------
 
-export interface SuccessResponse {
-    success: boolean;
-}
-
-export interface IdEntity {
-    id: string;
-}
+export interface SyncMetadata_aircall_users {
+};
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)